ValveResourceFormat/Resource/ResourceTypes/ModelAnimation/Animation.cs

```diff
@@ -90,38 +91,62 @@ public static IEnumerable<Animation> FromData(IKeyValueCollection animationData,
                     container.Length - (int)containerReader.BaseStream.Position
                 );
 
+                var remapTable = localChannel.RemapTable
+                    .Select(i => Array.IndexOf(elements, i))
+                    .ToArray();
+                var wantedElements = remapTable.Where(boneID => boneID != -1).ToArray();
+                remapTable = remapTable
+                    .Select((boneID, i) => (boneID, i))
+                    .Where(t => t.boneID != -1)
+                    .Select(t => t.i)
+                    .ToArray();
+                var channelAttribute = localChannel.ChannelAttribute switch
+                {
+                    "Position" => AnimationChannelAttribute.Position,
+                    "Angle" => AnimationChannelAttribute.Angle,
+                    "Scale" => AnimationChannelAttribute.Scale,
+                    _ => AnimationChannelAttribute.Unknown,
+                };
+
+                if (channelAttribute == AnimationChannelAttribute.Unknown)
+                {
+                    Console.WriteLine($"Unknown channel attribute '{localChannel.ChannelAttribute}' encountered with '{decoder}' decoder");
```

Comment:
This now spams `Unknown channel attribute 'data'`, that's what the `case "data"` ignore was for.